templates/index.html

```diff
@@ -4,162 +4,30 @@
-            font-size: 80px;
-            letter-spacing: 2px;
-            font-family: fantasy;
-            background: linear-gradient(
-            to right,
-            rgba(249,105,14,1) 0%,
-            rgba(102,51,153,1) 100%
-            );
-            -webkit-background-clip: text;
-            -webkit-text-fill-color: transparent;
-        }
-        .formbox{
-            background: rgba(0, 0, 0, 0.5);
-            padding: 30px;
-            border-radius: 10px;
-        }
-
-        #snackbar {
-          visibility: hidden;
-          margin-left: -125px;
-          background-color: black;
-          color: #fff;
-          text-align: center;
-          border-radius: 10px;
-          padding: 16px;
-          position: fixed;
-          z-index: 1;
-          left: 80%;
-          bottom: 30px;
-          font-size: 15px;
-        }
-
-        #snackbar.show {
-          visibility: visible;
-          -webkit-animation: fadein 0.5s, fadeout 0.5s 10s;
-          animation: fadein 0.5s, fadeout 0.5s 10s;
-        }
-
-        @-webkit-keyframes fadein {
-          from {bottom: 0; opacity: 0;} 
-          to {bottom: 30px; opacity: 1;}
-        }
-
-        @keyframes fadein {
-          from {bottom: 0; opacity: 0;}
-          to {bottom: 30px; opacity: 1;}
-        }
-
-        @-webkit-keyframes fadeout {
-          from {bottom: 30px; opacity: 1;} 
-          to {bottom: 0; opacity: 0;}
-        }
-
-        @keyframes fadeout {
-          from {bottom: 30px; opacity: 1;}
-          to {bottom: 0; opacity: 0;}
-        }
-
-      .autocomplete {
-        position: relative;
-        display: inline-block;
-      }
-
-      .autocomplete-items {
-        position: absolute;
-        border: 1px solid #d4d4d4;
-        border-bottom: none;
-        border-top: none;
-        z-index: 99;
-        /*position the autocomplete items to be the same width as the container:*/
-        top: 100%;
-        left: 0;
-        right: 0;
-      }
+   
 
-      .autocomplete-items div {
-        padding: 10px;
-        cursor: pointer;
-        background-color: #0c0c0d; 
-        border-bottom: 1px solid #d4d4d4; 
-      }
-
-      /*when hovering an item:*/
-      .autocomplete-items div:hover {
-        background-color: DodgerBlue; 
-      }
-
-      /*when navigating through the items using the arrow keys:*/
-      .autocomplete-active {
-        background-color: DodgerBlue !important; 
-        color: #0c0c0d; 
-      }
-
-      @media (max-width: 665px)
-        {
-            .title{
-                font-size: 45px;
-            }
-        }
-        @media (max-width: 400px)
-        {
-            .title{
-                font-size: 40px;
-            }
-        }
-
-
-	</style>
+       
 	   
+<script>
+  function myFunction() {
+    var x = document.getElementById("snackbar");
+    x.className = "show";
+    setTimeout(function(){ x.className = x.className.replace("show", ""); }, 20000);
+  }
+</script>
```

Comment:
if possible add this also in external js file

Reply:
ok